kitsune/questions/templates/questions/question_details.html

```diff
@@ -118,7 +118,9 @@ <h2 class="summary" itemprop="headline">{{ question.title }}</h2>
         </div>
         <div class="asked-on">
           {{ _('Posted') }}<br>
-          <time itemprop="dateCreated" datetime="{{ question.created }}">{{ datetimeformat(question.created) }}</time>
+          <span itemprop="dateCreated">
+            {{ datetimeformat(question.created) }}
+          </span>
```

Comment:
yeah sorry my bad, i see that now. basically we need that `itemprop` attribute to be on the `<time>`. so perhaps `datetimeformat` will need to be updated. i need some time to dig around and figure out what should happen here.